config/sankey_csv.yml

```diff
@@ -1436,7 +1497,46 @@ rows:
     Carrier: Liquid hydrogen
     Category: Export
     Type: Input
-    Value: imported_liquid_hydrogen_to_export_in_sankey  
-
+    Value: imported_liquid_hydrogen_to_export_in_sankey
 
+  - Group: Primary demand
+    Carrier: Non-biogenic waste
+    Category: Non-biogenic waste
+    Type: Output
+    Value: total_primary_demand_of_non_biogenic_waste
+  - Group: Conversion
+    Carrier: Non-biogenic waste
+    Category: Non-biogenic waste for blending
+    Type: Input
+    Value: non_biogenic_waste_to_blending_in_sankey
```

Comment:
@kndehaan shouldn't there also be conversion input of biogenic waste to result in the waste blended waste mix?

Reply:
Yes, it's also in the file under the biomass carrier:
```
- Group: Conversion
    Carrier: Biomass
    Category: Biogenic waste for blending
    Type: Input
    Value: biomass_products_to_blending_in_sankey
```